pkgs/development/tools/unityhub/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, appimageTools, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "unityhub";
+  version = "2.2.2";
+in
+
+appimageTools.wrapType2 rec {
+  name = "${pname}-${version}";
+
+  extraPkgs = (pkgs: with pkgs; with xorg; [ gtk2 gdk_pixbuf glib libGL libGLU nss nspr
+    alsaLib cups gnome2.GConf libcap fontconfig freetype pango
+    cairo dbus dbus-glib libdbusmenu libdbusmenu-gtk2 expat zlib libpng12 udev tbb
+    libpqxx gtk3 libsecret lsb-release openssl nodejs ncurses5
+
+    libX11 libXcursor libXdamage libXfixes libXrender libXi
+    libXcomposite libXext libXrandr libXtst libSM libICE libxcb ]);
+
+  profile = ''
+    export XDG_DATA_DIRS=${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS
+  '';
+
+  src = fetchurl {
+    url = "https://public-cdn.cloud.unity3d.com/hub/prod/UnityHub.AppImage";
```

Comment:
I just realized, there will be broken hashes:
https://aur.archlinux.org/packages/unityhub/

Reply:
for posterity


> Unity doesn't provide version-specific urls for Unity Hub so on every update there will be some period where the sum is incorrect until I find time to update the package.
> 
> Then even after I update the package, the CDN will keep serving the old version for a while to some people...
> 
> There isn't much that I can do to make this better, so please be patient.

Reply:
The alternative would be to upload UnityHub.AppImage to archive.org.
The hash is broken by the way in master.